packages/asc-ui/src/components/Downloader/Downloader.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import Downloader from './Downloader'
+
+const DownloaderComponent: React.FC<{}> = () => (
+  <>
```

Comment:
fragment?

Reply:
no, this notation is overall used in this project